components/lcd/esp_lcd_gc9503/README.md

```diff
@@ -19,63 +19,84 @@ You can add them to your project via `idf.py add-dependancy`, e.g.
 
 Alternatively, you can create `idf_component.yml`. More is in [Espressif's documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-component-manager.html).
 
-## Example use
+## Initialization Code
 
-Create 3-wire SPI panel IO using [esp_lcd_panel_io_additions](https://components.espressif.com/components/espressif/esp_lcd_panel_io_additions) component.
-
-```
+```c
+    // Using component esp_lcd_panel_io_additions: https://components.espressif.com/components/espressif/esp_lcd_panel_io_additions
+    ESP_LOGI(TAG, "Install 3-wire SPI panel IO");
     spi_line_config_t line_config = {
```

Comment:
This type is defined in [esp_lcd_panel_io_additions](https://components.espressif.com/components/espressif/esp_lcd_panel_io_additions?language=en), right? Could we add a note that this component must be explicitely added to user's project?

(since it is not liste in `idf_component.yml` of this component?)

Reply:
Yeah, you're right. Already done.